Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -957,6 +957,22 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
 
         // expect this change stream to have more events after resuming
         expect(try changeStream2.nextWithTimeout()).toNot(beNil())
+
+        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 2) else {
+            print("Skipping test case for server version \(try client.serverVersion())")
+            return
+        }
+
+        // test with startAfter
+        let newColl = try coll.renamed(to: self.getCollectionName(suffix: "2"))
+
+        _ = try changeStream2.nextWithTimeout()
+        expect(try changeStream2.nextWithTimeout()?.operationType).to(equal(.invalidate))
+
+        let opts = ChangeStreamOptions(startAfter: changeStream2.resumeToken)
+
+        // resuming (with startAfter) after an invalidate event should work
+        expect(try newColl.watch(options: opts)).toNot(throwError())
```

Comment:
I think `startAfter` is used to essentially resume a change stream on the original collection, even after that collection may have been invalidated. So I think we want to use `coll.watch` here. Also, I think we should perform an insert or something into `coll` and ensure the new change stream resumed via `startAfter` picks up the event.